<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolarPowerDialog</class>
 <widget class="QDialog" name="SolarPowerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTip">
   <string>Add in preset average oktas to monthly calculations.</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>30</y>
     <width>467</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Solar Power System Simulator</string>
   </property>
  </widget>
  <widget class="QComboBox" name="computationComboBox">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>105</y>
     <width>221</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>85</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Computation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="latitudeLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Location on the earth surface of the installation.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>-34.929</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="moduleAngleLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Angle of the solar modules to the vertical.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>90</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="moduleOffsetLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Angle offset of the module from North towards East.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="costLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;tariff ($/kwH) paid by the user.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>0.18</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="feedInLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;tariff ($/kwH) paid to the user.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>0.50</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Location on the earth surface of the installation.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Latitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Angle of the solar modules to the vertical.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Module Angle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Angle offset of the module from North towards East.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Module Offset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;tariff ($/kwH) paid by the user.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;tariff ($/kwH) paid to the user.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Feed in</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usageLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;average power in kW taken by the user during the day.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;average power in kW taken by the user during the day.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Usage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Angle of the sun's rays to the equatorial plane (only used for day computations).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Declination</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="declinationLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Angle of the sun's rays to the equatorial plane (only used for day computations).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>23.45</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="goPushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Location on the earth surface of the installation.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Environment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>340</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Number of cells in series. Leave at 1 if cell resistance is zero. If greater than 1, voltages above must be divided by number of cells.</string>
   </property>
   <property name="text">
    <string>No of Cells</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>310</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Voltage at maximum power point from datasheets</string>
   </property>
   <property name="text">
    <string>Max P Voltage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>250</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Open circuit voltage from datasheets</string>
   </property>
   <property name="text">
    <string>O/C Voltage</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberModulesLineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Number of modules in the system</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="ocVoltageLineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Open circuit voltage from datasheets</string>
   </property>
   <property name="text">
    <string>22.1</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Location on the earth surface of the installation.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Modules</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>190</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Efficiency of electronics converting output to standard power levels.</string>
   </property>
   <property name="text">
    <string>Efficiency</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scCurrentLineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Short circuit current from datasheets.</string>
   </property>
   <property name="text">
    <string>8.02</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Number of modules in the system</string>
   </property>
   <property name="text">
    <string>No of Modules</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberCellsLineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Number of cells in series. Leave at 1 if cell resistance is zero. If greater than 1, voltages above must be divided by number of cells.</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>280</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>datasheets</string>
   </property>
   <property name="text">
    <string>Max P Current</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="efficiencyLineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>190</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Efficiency of electronics converting output to standard power levels.</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxPCurrentLineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Current at maximum power point from datasheets</string>
   </property>
   <property name="text">
    <string>7.23</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxPVoltageLineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Voltage at maximum power point from datasheets</string>
   </property>
   <property name="text">
    <string>17.3</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>220</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Short circuit current from datasheets.</string>
   </property>
   <property name="text">
    <string>S/C Current</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="computationProgressBar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>365</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberCellsLineEdit_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Resistance os a cell. This is not normally provided and should be left at zero.</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>370</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Resistance os a cell. This is not normally provided and should be left at zero.</string>
   </property>
   <property name="text">
    <string>Cell resistance</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="oktaCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Cloud cover              </string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>latitudeLineEdit</tabstop>
  <tabstop>declinationLineEdit</tabstop>
  <tabstop>moduleAngleLineEdit</tabstop>
  <tabstop>moduleOffsetLineEdit</tabstop>
  <tabstop>costLineEdit</tabstop>
  <tabstop>feedInLineEdit</tabstop>
  <tabstop>usageLineEdit</tabstop>
  <tabstop>numberModulesLineEdit</tabstop>
  <tabstop>efficiencyLineEdit</tabstop>
  <tabstop>scCurrentLineEdit</tabstop>
  <tabstop>ocVoltageLineEdit</tabstop>
  <tabstop>maxPCurrentLineEdit</tabstop>
  <tabstop>maxPVoltageLineEdit</tabstop>
  <tabstop>numberCellsLineEdit</tabstop>
  <tabstop>computationComboBox</tabstop>
  <tabstop>goPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
